src/exporters/cds.py

```diff
@@ -198,8 +198,13 @@ def _correct_input(value, key):
         return str(value)
 
     @staticmethod
-    def _check_iterable(value, key):
-        pass
+    def _check_iterable(self, value, key):
```

Comment:
@gabrieltseng I need some help here

The alternative is:
`_check_iterable(value, key):`

Without the `self` argument in the `_check_iterable` function, testing the function [in the notebook](https://github.com/esowc/ml_drought/blob/26bf1e76ff0f3259715eadf9f42a3ae3a466d52f/notebooks/draft/01_tl_initial_test.ipynb) fails:
```
TypeError: _check_iterable() takes 2 positional arguments but 3 were given
```

So I wonder how the `@staticmethod` actually works and why pytest passes but the notebook fails?

The function is there to catch the errors that the user provides only an integer for a month for example